distributed/client.py

```diff
@@ -602,9 +602,28 @@ def set_error(self, exception, traceback):
             The exception
         traceback: Exception
             The traceback
+        erred_on: set
+            Workers where it errored
+        key: tuple
+            Task key
         """
         _, exception, traceback = clean_exception(exception, traceback)
 
+        if (
+            dask.config.get("admin.traceback.shorten", False) is True
```

Comment:
As a rule of thumb, it's a very bad idea to pass a default to dask.config.get.
With my changes in dask/dask, this test is always false. Could you amend this PR to match?

Reply:
Updated. About the default value, can you clarify, if the setting is added in `dask`, and not in `distributed`, how can I avoid using the default? Or should I add the same setting on both sides?

Reply:
Note: every version of distributed pins the matching version of dask, and vice versa.